compiler/cbmc/src/irep/to_json.rs

```diff
@@ -11,7 +12,11 @@ impl ToJson for Irep {
             output.insert("sub".to_string(), self.sub.to_json());
         }
         if !self.named_sub.is_empty() {
-            output.insert("namedSub".to_string(), self.named_sub.to_json());
+            let mut d = BTreeMap::new();
```

Comment:
You can use `BTreeMap::FromIter()` here to avoid the loop

Reply:
Yeah, that required a map though so it seemed like more complicated code